src/main/java/org/apache/sysds/runtime/controlprogram/federated/FederatedRequest.java

```diff
@@ -33,20 +35,24 @@
 	
 	private FedMethod _method;
 	private List<Object> _data;
+	private boolean checkPrivacy;
```

Comment:
could we prepend it with `_` so the variables have the same format?

Reply:
Good idea!